File: FindWeather/lab08/WeatherData.java
public class WeatherData {
    String temperature;
     String condition;
     String source;

    public WeatherData(String temperature, String condition, String source) {
        this.temperature = temperature;
        this.condition = condition;
        this.source = source;
    }

    @Override
    public String toString() {
        return "Source: " + source + ", Temperature: " + temperature + "°C, Condition: " + condition;
    }
}

File: FindWeather/Lab-8_FindWeather/WeatherData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab_8_FindWeather
{
    public class WeatherData
    {
        public double Temperature { get; set; } 
        public string Conditions { get; set; }
        public string Location { get; set; }
        public string Source { get; set; }

        public override string ToString()
        {
            return $"Weather in {Location} from {Source}: {Temperature}°C, {Conditions}";
        }
    }
}
